.github/workflows/test.yml

```diff
@@ -102,7 +107,10 @@ jobs:
           maturin list-python
           PYTHON_INTERPRETER=$(maturin list-python | grep -o "CPython $(cat .python-version | grep -o -E '^[^\.]+\.[^\.]+').* at .*" | cut -d' ' -f 4 | tr '\\' '/')
           echo "Selected interpreter: ${PYTHON_INTERPRETER}"
-          just build-all "${{ matrix.target.rust-target }}"
+          LLVM_SYS_100_PREFIX=${LLVM_DIR} just build-all "${{ matrix.target.rust-target }}"
+        env:
+          LLVM_DIR: ${{ github.workspace }}/llvm
```

Comment:
You can declare `LLVM_SYS_100_PREFIX` here I guess (rather than before executing `just build-all`).